src/components/maincontroller.ts

```diff
@@ -725,7 +702,12 @@ export function createMediaInformation(
         );
     }
 
-    mediaInfo.customData.startPositionTicks = streamInfo.startPosition || 0;
+    // wtf this breaks everything
+    // mediaInfo.customData.startPositionTicks = streamInfo.startPosition || 0;
+    // try instead:
+    // if(streamInfo.startPosition)
+    //     mediaInfo.customData.startPositionTicks = streamInfo.startPosition
+    // however this is likely to be unneeded
```

Comment:
Same comment thing

Reply:
I think I used this comment to discuss it with @YouKnowBlom because I didn't understand where startPosition would be set from, but theoretically the client can set it. I think we may need the second part for resolving a suspected bug. But would need to look over the jellyfin-web code to understand this better.